src/Handler.php

```diff
@@ -104,18 +105,45 @@ private function processOperations(array $operations, array $versions, bool $red
 
         foreach ($operations as $offset => $operation) {
             $sql = $this->getBuilder()->build($operation);
+
+            // Determine if transaction is required.
+            $transactionRequired = $operations instanceof ExecOperation;
```

Comment:
This line should be performing `instanceof` against `$operation` instead of the array.

Reply:
@joshmcrae lol, yes... yes it should... thanks for that.

Reply:
Resolved!